official/projects/longformer/longformer_encoder.py

```diff
@@ -0,0 +1,394 @@
+# Copyright 2021 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+Longformer encoder. Modified From huggingface/transformers
+"""
+
+# pylint: disable=g-classes-have-attributes
+
+from typing import Any, Callable, Optional, Union, List
+from absl import logging
+import tensorflow as tf
+
+from official.nlp.modeling import layers
+from official.projects.longformer.longformer_encoder_block import LongformerEncoderBlock
+
+def shape_list(tensor: tf.Tensor) -> List[int]:
```

Comment:
Making it private or can we simply use tf_utils.get_shape_list? https://github.com/tensorflow/models/blob/34a93745259ca40db718763a85ad38fc8ee433e5/official/modeling/tf_utils.py#L131